covid-19.md

```diff
@@ -6,29 +6,34 @@ title: Extra Support for COVID-19 Research
 OSG Support for COVID-19 Research
 =================================
 
-Like many others in the research computing community, OSG is helping to organize
-extra resources to be made available to COVID-19 research!  These resources include:
+Please contact us at <mailto:help@opensciencegrid.org> if your research group could benefit
+from computing resources or consulting related to COVID-19, or if you would like to support COVID-19
+research by sharing your own capacity.
+
+OSG is helping to organize extra resources to be made available to COVID-19 research!  These resources include:
 
-- **Consultation and facilitation** for quickly scaling up and automating workflows.
-- **Expertise** on using high throughput computing for maximizing research throughput.
-- **Access to computing capacity** through the [OSG Connect](https://support.opensciencegrid.org/support/home)
-  service which provides a batch system submit environment based on HTCondor.  The OSG aggregates
+- **Proactive consultation and facilitation** for quickly scaling up and automating workloads with up to [thousands of 
+concurrently-running jobs](https://gracc.opensciencegrid.org/dashboard/db/covid-19-research?orgId=1).
+- **Expertise** on using high throughput computing for maximizing research throughput, perhaps including 
+development effort.
+- **Access to computing capacity** through the *[OSG Connect](https://support.opensciencegrid.org/support/home)*
+  service which provides a batch system submit environment based on HTCondor (allowing up to millions of jobs submitted via a
+  single submit file).  The OSG aggregates
   about 100 clusters across the US and COVID-19 research could access tens of thousands of cores
   through this mechanism.  The OSG also provides a CDN for distributing data to jobs.
+- **Capabilities for a variety of computational research tasks.** See [Computation on the OSG](https://opensciencegrid.org/about/computation/).
 
+Sites Supporting COVID-19 Research
```

Comment:
Maybe this should be "Submitting to Sites Supporting COVID-19 Research"? The current headline makes it seem like I'm about to get a list of sites, and then the paragraph started talking about researchers, and I got confused.